app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -491,11 +491,11 @@ public boolean onMenuItemClick(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.disable_sleeptimer_item: // Fall-through
             case R.id.set_sleeptimer_item:
-                new SleepTimerDialog().show(getFragmentManager(), "SleepTimerDialog");
+                new SleepTimerDialog().show(getParentFragmentManager(), "SleepTimerDialog");
```

Comment:
Why parent and not child fragment manager? As I understood the fragment manager, this (and some others in this commit) should use the child fragment manager. I have not tried it, though.

Reply:
Very well.